src/workerd/api/blob.h

```diff
@@ -14,10 +14,8 @@ class ReadableStream;
 // An implementation of the Web Platform Standard Blob API
 class Blob: public jsg::Object {
 public:
-  Blob(kj::Array<byte> data, kj::String type)
-      : ownData(kj::mv(data)), data(ownData.get<kj::Array<byte>>()), type(kj::mv(type)) {}
```

Comment:
Why de-inline trivial constructors like this? I generally expect the compiler is pretty good at optimizing this sort of thing, but can't anymore if it's not inline...